docs/nidcpower/class.rst

```diff
@@ -858,6 +858,50 @@ get_channel_name
 
 
 
+get_channel_names
+-----------------
+
+    .. py:currentmodule:: nidcpower.Session
+
+    .. py:method:: get_channel_names(indices)
+
+            Returns a list of channel names for given channel indices.
+
+            
+
+
+            .. tip:: This method requires repeated capabilities. If called directly on the
+                nidcpower.Session object, then the method will use all repeated capabilities in the session.
+                You can specify a subset of repeated capabilities using the Python index notation on an
+                nidcpower.Session repeated capabilities container, and calling this method on the result.
+
+
+            :param indices:
+
+
+                Index list for the channels in the session. Valid values are from zero to the total number of channels in the session minus one. The index string can be one of the following formats:
+
+                -   A comma-separated list—for example, "0,2,3,1"
+                -   A range using a hyphen—for example, "0-3"
+                -   A range using a colon—for example, "0:3 "
```

Comment:
I'm guessing this came from CVI? so it doesn't say you can use iterable or anything like that.
We should consider updating for the Python API. If it needs to be in separate PR please open issue.

Reply:
[I put that text in hapigen](https://ni.visualstudio.com/DevCentral/_git/ni-central/pullrequest/139277?_a=files&path=%2Fsrc%2Fnidcpower%2FnidcpowerBaseDistribution%2FnidcpowerAPIMetadata%2Fsource%2FnidcpowerAPIMetadata%2Fnimi_python_documentation%2Ffunctions%2Ffunctions.hapigen), because I was trying to match what I saw in the [niDigital version of the function](https://github.com/ni/nimi-python/blob/master/docs/nidigital/class.rst#get_channel_names). Let me know if you still want me to create an issue.

Reply:
I agree we should update it and I think it should be in a separate PR where we update both nidcpower and nidigital.

Reply:
Created issue #1595.